docs/guides/qiskit-addons-sqd-get-started.ipynb

```diff
@@ -0,0 +1,2012 @@
+    "MAX_DAVIDSON_CYCLES = 200"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d977f0e1-0e21-45d0-8971-92e1773f418b",
+   "metadata": {},
+   "source": [
+    "Next, in order to plot the convergence, define arrays to store the approximation of the ground state energy, expectation value of the $\\langle S \\rangle ^2$, and the orbital occupancy of the molecule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "a02aaca2-bca2-4531-b49a-3718a37de948",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Self-consistent configuration recovery loop\n",
+    "energy_hist = np.zeros((ITERATIONS, NUM_BATCHES))  # energy history\n",
+    "spin_sq_hist = np.zeros((ITERATIONS, NUM_BATCHES))  # spin history\n",
+    "occupancy_hist = np.zeros((ITERATIONS, 2 * num_orbitals))\n",
+    "occupancies_bitwise = (\n",
+    "    None  # orbital i corresponds to column i in bitstring matrix\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05823f66-cf57-43e0-bd9a-a77c6b6f8786",
+   "metadata": {},
+   "source": [
+    "Now, run the configuration recovery loop. Each loop consists of three steps:\n",
+    "\n",
+    "1. Use the `recover_configurations()` method to obtain a refined bitstring matrix and probability array based on the average orbital occupancy.\n",
+    "1. The `postselect_and_subsample()` function is used to collect batches of subsamples to diagonalize over.\n",
+    "1. The batches of subsamples are then used as arguments to the `solve_fermion()` method to obtain an approximation of the ground state."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "a6d492b5-d86b-43cd-9171-4add0903fe84",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starting configuration recovery iteration 0\n",
+      "Starting configuration recovery iteration 1\n",
+      "Starting configuration recovery iteration 2\n",
+      "Starting configuration recovery iteration 3\n",
+      "Starting configuration recovery iteration 4\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_addon_sqd.configuration_recovery import recover_configurations\n",
+    "from qiskit_addon_sqd.subsampling import postselect_and_subsample\n",
+    "from qiskit_addon_sqd.fermion import flip_orbital_occupancies, solve_fermion\n",
+    "\n",
+    "for i in range(ITERATIONS):\n",
+    "    print(f\"Starting configuration recovery iteration {i}\")\n",
+    "    # On the first iteration, we have no orbital occupancy information from the\n",
+    "    # solver, so we just post-select from the full bitstring set based on hamming weight.\n",
+    "    if occupancies_bitwise is None:\n",
+    "        bitstring_matrix_tmp = bitstring_matrix_full\n",
+    "        probs_array_tmp = probs_array_full\n",
+    "\n",
+    "    # If we have average orbital occupancy information, we use it to refine the full set of noisy configurations\n",
```

Comment:
```suggestion
    "    # If there is average orbital occupancy information, use it to refine the full set of noisy configurations\n",
```

Reply:
I know I left "we" in the previous comment, but I get rid of it when there's a straightforward way to do so. :)